signac/contrib/import_export.py

```diff
@@ -91,20 +147,52 @@ def format_field(self, value, format_spec):
 
 
 class _SchemaPathEvaluationError(RuntimeError):
+    """Raised for errors in schema path evaluation."""
     pass
 
 
 def _make_path_function(jobs, path):
-    "Generate a path function for jobs or use `path` if its a callable."
+    """Generate a path function for jobs or use ``path`` if it is callable.
+
+    Parameters
+    ----------
+    jobs : iterable of :class:`~signac.contrib.job.Job`
+        A sequence of jobs (instances of :class:`~signac.contrib.job.Job`).
+    path : callable
+        A callable path generating function.
+
+    Returns
+    -------
+    callable
+        Path function for given job and path.
+
+    Raises
+    ------
+    ValueError
+        The path argument must either be ``None``, ``False``, or of type ``str``.
 
+    """
     if path is None:
         # Generate a path function based on the schema detected for jobs.
         path_function = _make_schema_based_path_function(jobs=jobs)
 
     elif path is False:
 
-        # Just use the job-id as path.
+        # Just use the job id as path.
         def path_function(job):
+            """Create path function when path is not given.
```

Comment:
I'm not sure that we really need to document these internal closures, but if we do then this doc-string is not correct.
```suggestion
            """Use job-id to construct path.
```

Reply:
@csadorf `pydocstyle` does require us to document internal closures, anything with a `def`. Thanks for the suggestion.